lib/features/cli/networking/create_client_session.dart

```diff
@@ -0,0 +1,15 @@
+import '../../../core/networking/websockets/middleware/protocol_middleware.dart';
+import '../../../core/networking/websockets/session.dart';
+
+/// Creates a new session to the server using the [host] and [port].
+///
+/// The created [ClientSession] is a wrapper around a `WebSocket` connection.
+Future<ClientSession> createClientSession(String host, int port) async {
+  const String prefix = 'ws';
```

Comment:
Why not put the `prefix` right into the string below?

Reply:
I wanted it separate in case we need to change it for encrypted traffic.

Reply:
Makes sense - probably optimized away by the compiler anyway. Was just wondering